### PR TITLE
[ATIP-Android] Modify atip android save object info process

### PR DESCRIPTION
**Pull request by jacky-young**
- Modify atip android save object info process

### CODE REVIEW

**Comment by jacky-young** [comment]
@Honry please review it.

**Comment by Honry** [comment]
Please provide your UT result.

**Comment by jacky-young** [comment]
verified locally.

File: tools/atip/README.md [hunk 1]
```diff
@@ -401,10 +401,11 @@ def i_visit_url(context, url):
 @step(u'I edit view "{params_kw}" to input "{text}"')
 @step(u'I edit index {n:d} EditText to input "{text}"')
 @step(u'I compare view "{params_kw}" info "{info_name}" with "{except_result}"')
-@step(u'I compare saved object "{key1}" equal "{key2}" on info "{info_name}"')
-@step(u'I compare saved object "{key1}" unequal "{key2}" on info "{info_name}"')
+@step(u'I compare saved info "{key1}" equal "{key2}"')
```
Comment (Honry) [inline tools/atip/README.md]: s/equal/equals to

File: tools/atip/README.md [hunk 2]
```diff
@@ -401,10 +401,11 @@ def i_visit_url(context, url):
 @step(u'I edit view "{params_kw}" to input "{text}"')
 @step(u'I edit index {n:d} EditText to input "{text}"')
 @step(u'I compare view "{params_kw}" info "{info_name}" with "{except_result}"')
```
Comment (Honry) [inline tools/atip/README.md]: The view "{params_kw}" info "{info_name}" should be "{except_result}"
would be better.

File: tools/atip/README.md [hunk 3]
```diff
@@ -401,10 +401,11 @@ def i_visit_url(context, url):
...
-@step(u'I compare saved object "{key1}" equal "{key2}" on info "{info_name}"')
-@step(u'I compare saved object "{key1}" unequal "{key2}" on info "{info_name}"')
+@step(u'I compare saved info "{key1}" equal "{key2}"')
+@step(u'I compare saved info "{key1}" unequal "{key2}"')
```
Comment (Honry) [inline tools/atip/README.md]: ditto.

**Comment by Honry** [comment]
LGTM

**Pull request merged**